### PR TITLE
SEO-176893-Missing-Alt-Text-React

### PR DESCRIPTION
**Pull request by berylouma**
Hello, 

I have worked on the meta properties.

Regards,
Beryl

### CODE REVIEW

File: ej2-react/document-editor/getting-started.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ documentation: ug
 domainurl: ##DomainURL##
 ---
 
-# Getting Started
+# Getting Started with React Document editor component
```
Comment (Yvone-Atieno) [inline ej2-react/document-editor/getting-started.md:11]: Getting Started with React Document Editor

File: ej2-react/grid/getting-started.md [hunk 2]
```diff
@@ -1,14 +1,14 @@
...
-description:  Checkout and learn about Getting started with React Grid component of Syncfusion Essential JS 2 and more details.
+description: Checkout and learn about Getting started with React Grid component of Syncfusion Essential JS 2 and more details.
 control: Getting started 
 platform: ej2-react
 documentation: ug
 domainurl: ##DomainURL##
 ---
 
-# Getting started
+# Getting started with React Grid component
```
Comment (Yvone-Atieno) [inline ej2-react/grid/getting-started.md:11]: In H1 remove the word component. Check in all places

**Review by Yvone-Atieno — CHANGES_REQUESTED** [review]
Commented

File: ej2-react/grid/getting-started.md (hunk 2 above)
Comment (berylouma) [reply ej2-react/grid/getting-started.md]: Done

File: ej2-react/document-editor/getting-started.md (hunk 1 above)
Comment (berylouma) [reply ej2-react/document-editor/getting-started.md]: Updated

**Pull request closed without merging**